lib/page/house_wikidata.rb

```diff
@@ -12,21 +13,33 @@ def title
       "EveryPolitician: #{country.name} — #{house.name}"
     end
 
+    def house
+      country.legislature(house_slug)
+    end
+
     def country
       index.country(country_slug)
     end
 
-    def house
-      country.legislature(house_slug)
+    def people_with_wikidata
+      people_with_and_without_wikidata[:with]
     end
 
-    # TODO: we shouldn't be passing raw Popolo, only what's needed
-    def popolo
-      house.popolo
+    def people_without_wikidata
+      people_with_and_without_wikidata[:without]
     end
 
     private
 
     attr_reader :house_slug, :country_slug, :index
+
+    def popolo
+      house.popolo
+    end
+
+    def people_with_and_without_wikidata
+      @with_and_without_array ||= popolo.persons.partition(&:wikidata)
+      { with: @with_and_without_array[0], without: @with_and_without_array[1] }
```

Comment:
it seems a little needless to convert this to a hash here, rather than just calling `people_with_and_without_wikidata.first` or `.last` in the public methods.